packages/usa-form/src/styles/_usa-form.scss

```diff
@@ -17,41 +17,11 @@
     text-decoration: none;
   }
 
-  .usa-input,
-  .usa-range,
-  .usa-select,
-  .usa-textarea {
+  :where(.usa-input, .usa-textarea, .usa-select, .usa-range) {
```

Comment:
> [!note]
> Reduced specificity with `where()` here because this rule was causing .usa-form to override input width classes because it was more specific.